docs-2.0/3.ngql-guide/7.general-query-statements/6.show/14.show-stats.md

```diff
@@ -0,0 +1,71 @@
+# SHOW STATS
+
+The `SHOW STATS` statement shows the statistics of the graph space collected by the latest `STATS` job.
+
+The statistics list the following information:
+
+* The number of vertices and edges in the graph space
+* The number of vertices with each tag
+* The number of edges of each edge type
+
+## Prerequisites
+
+You have successfully run the `SUBMIT JOB STATS` statement in the graph space you want to collect statistics. For more information, see [SUBMIT JOB STATS](../../18.operation-and-maintenance-statements/4.job-statements).
+
+>**NOTE:** The result of the `SHOW STATS` statement is based on the last executed `SUBMIT JOB STATS` statement. If you want to update the result, run `SUBMIT JOB STATS` again.
+
+## Syntax
+
+```ngql
+SHOW STATS
+```
+
+## Example
+
+```ngql
+nebula> USE nba;
+Execution succeeded (time spent 1075/1646 us)
+
+--Start a `STATS` job.
```

Comment:
'--' is not available for comments. And even it can work, a space may be necessary between "--" and the comment. 

check it through the article.